### ISSUE TITLE
Adding leuven-theme to GNU ELPA

### ISSUE THREAD

**Issue by skangas** [fniessen/emacs-leuven-theme#96]
leuven-theme.el is shipped with Emacs, but it would be nice if users could get upgrades through GNU ELPA.  It should be easy to add it, if we agree that it's a good idea.

Fabrice, what do you think?

**Comment by fniessen** [comment]
Dear @skangas , are you explicitly making a difference between MELPA and ELPA?  Because it's already on MELPA... and via Git... so, in fact, you do have 3 versions - the last 2 ones with the same lifecycle, though...

**Comment by skangas** [comment]
Yes, they are different archives.  [GNU ELPA](https://elpa.gnu.org/) is configured in Emacs by default, whereas [MELPA](https://melpa.org/#/) is not.

(Note that ELPA properly refers only to the protocol that is used by both GNU ELPA and MELPA.)

**Comment by skangas** [comment]
On current master, we have this MELPA-inspired version in `leuven.el`:

```
;; Version: 20200513.1928
```

On GNU ELPA, we do not follow the versioning scheme; all packages have versions like, e.g. "0.1". Do you think we could change the versioning scheme of `leuven.el` to make it fit in better with the rest of the packages (and indeed with Emacs itself)?

If the answer is yes, I'm happy to add it to GNU ELPA.

PS. BTW, maybe it's worth updating the version of leuven.el that is on the `emacs-28` branch in any case?

**Comment by fniessen** [comment]
Dear @skangas ,

Yes, we could change the naming conventions for the versions - no worry.
Yes, also, the current version of leuven.el could be made part of emacs-28.

Are you planning to do (both of) this?

**Comment by skangas** [comment]
Hi @fniessen,

Sure, I can add a pull request with the new version number. Do you have any preferences for where to start, or does `1.0` sound good to you?
